tailwind.config.js

```diff
@@ -86,59 +86,33 @@ module.exports = {
     }),
 
     fontFamily: {
-      sans: [
-        'Source Sans Pro',
-        'ui-sans-serif',
-        'system-ui',
-        '-apple-system',
-        'BlinkMacSystemFont',
-        '"Segoe UI"',
-        'Roboto',
-        '"Helvetica Neue"',
-        'Arial',
-        '"Noto Sans"',
-        'sans-serif',
-      ],
-      systemSans: [
-        'ui-sans-serif',
-        'system-ui',
-        '-apple-system',
-        'BlinkMacSystemFont',
-        '"Segoe UI"',
-        'Roboto',
-        '"Helvetica Neue"',
-        'Arial',
-        '"Noto Sans"',
-        'sans-serif',
-      ],
-      mono: ['ui-monospace', 'Menlo', 'Monaco', 'Consolas', '"Liberation Mono"', '"Courier New"', 'monospace'],
-      systemMono: ['ui-monospace', 'Menlo', 'Monaco', 'Consolas', '"Liberation Mono"', '"Courier New"', 'monospace'],
-      grotesk: ['Space Grotesk'],
-      spaceMono: ['Space Mono'],
+      sans: ['Inter'],
```

Comment:
@gitstart-sourcegraph is it best practice to not have the fallback system fonts (ui-sans-serif, system-ui,...) in addition to our web fonts (Inter, Roboto Mono)? Please let me know if we should keep them in just in case.